mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.core/src/org/eclipse/mylyn/gitlab/core/GitlabJSonArrayOperation.java

```diff
@@ -0,0 +1,54 @@
+package org.eclipse.mylyn.gitlab.core;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.apache.http.Header;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpRequestBase;
+import org.eclipse.mylyn.commons.core.operations.IOperationMonitor;
+import org.eclipse.mylyn.commons.repositories.http.core.CommonHttpClient;
+import org.eclipse.mylyn.commons.repositories.http.core.CommonHttpResponse;
+
+import com.google.gson.JsonArray;
+
+public abstract class GitlabJSonArrayOperation extends GitlabOperation<JsonArray> {
+
+	public GitlabJSonArrayOperation(CommonHttpClient client, String urlSuffix) {
+		super(client, urlSuffix);
+	}
+	@Override
+	protected JsonArray execute(IOperationMonitor monitor) throws IOException, GitlabException {
+		JsonArray result = null;
+		HttpRequestBase request = createHttpRequestBase();
+		addHttpRequestEntities(request);
+		CommonHttpResponse response = execute(request, monitor);
+		result = processAndRelease(response, monitor);
+		Header[] lh = response.getResponse().getHeaders("Link");
```

Comment:
lh is not a  very informative variable name

Reply:
Done with next commit